packages/ember-codemod-remove-ember-css-modules/src/migration/ember-app/steps/import-styles/update-class.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/ban-ts-comment, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access */
```

Comment:
TODO: Investigate whether `@codemod-utils/cli` should disable these rules by default (if `@codemod-utils/ast-javascript` is installed), or `@codemod-utils/ast-javascript` should provide better types.